### ISSUE TITLE
Paddle 有 adam + Warmup 的优化器吗？

### ISSUE THREAD

**Issue by wa008** [PaddlePaddle/Paddle#41729]
### 需求描述 Feature Description

如题，对标 pytroch 的 Radam  https://pytorch.org/docs/stable/generated/torch.optim.RAdam.html?highlight=radam#torch.optim.RAdam

**Comment by Ligoml** [comment]
你好，暂时没有对应的api，可以简单描述一下你会在什么任务下用到这个api吗？
